packages/zip/src/exported.js

```diff
@@ -1,3 +1,7 @@
+// @ts-check
+
+export {};
```

Comment:
If switching to exported types, can we try to add the following to `zip/index.js` so that we can use import the types directly using package name?
```js
// eslint-disable-next-line import/export
export * from './src/exported.js';
```